tools/build.sh

```diff
@@ -13,7 +13,7 @@ while getopts ":l:m:d:p:" o; do
       ;;
     m)
       mcu=${OPTARG}
-      ((mcu == 'H' || mcu == 'L')) || usage
+      ((mcu == 'X' || mcu == 'H' || mcu == 'L')) || usage
```

Comment:
Please, if it works for you use " instead '. At this time I have found that Ubuntu 22.04 throws an error trying to compile with ' in MCU
```
      ((mcu == "X" || mcu == "H" || mcu == "L")) || usage
```

Reply:
I did change this, but out of interest, what kind of error are you getting with the singel quotes?

Reply:
It throws a syntax error:
```
blue@blue:~/proyectos/bluejay-bb5/tools$ ./build.sh -l G -m H -d 30 -p 48
./build.sh: línea 16: ((: mcu == 'X' || mcu == 'H' || mcu == 'L': error sintáctico: se esperaba un operando (el elemento de error es "'X' || mcu == 'H' || mcu == 'L'")
Usage: ./build.sh [-l <A-W>] [-m <H|L>] [-d <integer>] [-p <24|48|96>]
``

Reply:
Very interesting. In any case, I made the change. Did not make a difference on my end.